tests/main/interfaces-content/task.yaml

```diff
@@ -31,6 +31,10 @@ execute: |
     echo "And the current mount profile is the same as the desired mount profile"
     diff -u /run/snapd/ns/snap.test-snapd-content-plug.fstab /var/lib/snapd/mount/snap.test-snapd-content-plug.fstab
 
+    if [ "$(snap debug confinement)" = none ]; then
```

Comment:
Hold on, the content interface uses mount backend so it should work everywhere. There is some apparmor too but it doesn't make it any less effective when we unmount things.